src/form/index.es

```diff
@@ -165,21 +189,42 @@ class Form extends PureComponent {
       'buttonClass',
       'alternativeAction',
       'locked',
+      'defaultLocked',
     );
 
+    const Component = as || 'form';
+    const isNativeForm = Component === 'form';
+
+    const formProps = { className };
+    if (isNativeForm) {
+      formProps.method = 'post';
+      formProps.onSubmit = this.submit;
+      formProps.noValidate = true;
+    }
+
     let error;
     if (formError) {
       error = <strong className="c-form-error ui-error">{formError}</strong>;
     }
 
     let button;
     if (buttonText) {
-      const buttonClassName = buttonClass || 'ui-button ui-button-primary';
-      button = (
-        <button className={buttonClassName} type="submit" disabled={this.isDisabled()}>
-          {buttonText}
-        </button>
-      );
+      const ButtonComponent = isNativeForm ? 'button' : 'span';
+
+      const buttonProps = {
+        className: clsx(buttonClass || 'ui-button ui-button-primary', {
+          'is-disabled': isDisabled,
+        }),
+      };
+
+      if (isNativeForm) {
+        buttonProps.type = 'submit';
+        buttonProps.disabled = isDisabled;
+      } else {
+        buttonProps.onClick = this.submit;
+      }
+
+      button = <ButtonComponent {...buttonProps}>{buttonText}</ButtonComponent>;
```

Comment:
move to separate render function? or maybe component?

Reply:
+1 for the render function